app/views/surveys/component/_form-footer.pug

```diff
@@ -7,6 +7,10 @@ section.form-footer
             i.icon.icon-check= ' '
             span(data-i18n='formfooter.validate.btn')= t('formfooter.validate.btn')
         else if type === 'fieldsubmission' || type === 'view'
```

Comment:
nextPrompt may require an additional check to make sure it is not added in views that should not support it (even if protection has been added to Enketo's OC API). Tbc, depends on Paul's answer.

Reply:
updated, support on preview too